ui/src/main/webapp/package.json

```diff
@@ -26,118 +26,118 @@
-    "@angular/common": "^2.4.5",
-    "@angular/compiler": "^2.4.5",
-    "@angular/compiler-cli": "^2.4.5",
-    "@angular/core": "^2.4.5",
-    "@angular/forms": "^2.4.5",
-    "@angular/http": "^2.4.5",
-    "@angular/platform-browser": "^2.4.5",
-    "@angular/platform-browser-dynamic": "^2.4.5",
-    "@angular/platform-server": "^2.4.5",
-    "@angular/router": "^3.4.5",
+    "@angular/common": "^2.4.9",
+    "@angular/compiler": "^2.4.9",
+    "@angular/compiler-cli": "^2.4.9",
+    "@angular/core": "^2.4.9",
+    "@angular/forms": "^2.4.9",
+    "@angular/http": "^2.4.9",
+    "@angular/platform-browser": "^2.4.9",
+    "@angular/platform-browser-dynamic": "^2.4.9",
+    "@angular/platform-server": "^2.4.9",
+    "@angular/router": "^3.4.9",
     "@swimlane/ngx-charts": "^4.2.1",
-    "angular2-moment": "^1.1.0",
+    "angular2-moment": "^1.3.0",
     "bootstrap": "3.3.7",
     "bootstrap-datepicker": "1.6.4",
     "bootstrap-select": "1.12.2",
-    "bootstrap-switch": "3.3.3",
+    "bootstrap-switch": "3.3.4",
     "bootstrap-touchspin": "3.1.1",
     "bootstrap-treeview": "1.2.0",
     "c3": "0.4.11",
     "core-js": "^2.4.1",
-    "d3": "4.4.0",
-    "d3-selection-multi": "1.0.0",
-    "d3-transition": "1.0.3",
-    "datatables": "1.10.9",
+    "d3": "4.7.2",
+    "d3-selection-multi": "1.0.1",
+    "d3-transition": "1.0.4",
+    "datatables": "1.10.13",
     "datatables-colreorder": "1.2.0",
     "drmonty-datatables-colvis": "1.1.2",
-    "font-awesome": "4.3.0",
+    "font-awesome": "4.7.0",
     "google-code-prettify": "^1.0.5",
     "humanize-duration-ts": "^1.0.1",
     "intl": "^1.2.5",
     "jquery": "^3.1.1",
-    "jquery-match-height": "0.7.0",
-    "jstree": "^3.3.2",
-    "karma-junit-reporter": "^1.1.0",
-    "ng2-file-upload": "^1.1.0",
-    "patternfly": "^3.16.0",
+    "jquery-match-height": "0.7.2",
+    "jstree": "^3.3.3",
+    "karma-junit-reporter": "^1.2.0",
+    "ng2-file-upload": "^1.2.0",
+    "patternfly": "^3.22.1",
     "patternfly-bootstrap-combobox": "1.1.7",
-    "prismjs": "1.5.1",
-    "reflect-metadata": "^0.1.3",
-    "rxjs": "5.0.1",
-    "showdown": "^1.5.1",
+    "prismjs": "1.6.0",
+    "reflect-metadata": "^0.1.10",
+    "rxjs": "5.2.0",
+    "showdown": "^1.6.4",
     "text-loader": "^0.0.1",
-    "webpack-merge": "^0.15.0",
-    "zone.js": "^0.7.6"
+    "webpack-merge": "^4.0.0",
+    "zone.js": "^0.8.0"
   },
   "devDependencies": {
-    "@angularclass/hmr": "^1.0.1",
+    "@angularclass/hmr": "^1.2.2",
     "@angularclass/hmr-loader": "^3.0.2",
-    "@ngtools/webpack": "^1.2.10",
+    "@ngtools/webpack": "^1.2.12",
     "@types/bootstrap": "^3.3.32",
-    "@types/core-js": "^0.9.35",
-    "@types/d3": "4.4.1",
-    "@types/d3-array": "0.0.3",
-    "@types/d3-brush": "0.0.3",
-    "@types/d3-color": "0.0.3",
-    "@types/d3-format": "0.0.3",
-    "@types/d3-interpolate": "0.0.4",
-    "@types/d3-scale": "0.0.3",
-    "@types/d3-selection": "0.0.3",
+    "@types/core-js": "^0.9.37",
```

Comment:
@OndraZizka It seems updating corejs types broke it. See https://github.com/DefinitelyTyped/DefinitelyTyped/issues/15140